src/Task/PhpunitBridge.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace GrumPHP\Task;
+
+use GrumPHP\Runner\TaskResult;
+use GrumPHP\Task\Context\ContextInterface;
+use GrumPHP\Task\Context\GitPreCommitContext;
+use GrumPHP\Task\Context\RunContext;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+
+/**
+ * PhpunitBridge task
+ *
+ * @link https://symfony.com/doc/current/components/phpunit_bridge.html
+ */
+class PhpunitBridge extends AbstractExternalTask
```

Comment:
This task is not registered in the service config file located at `resources/config/tasks.yml`